dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/trackedentity/RequestParams.java

```diff
@@ -98,7 +98,7 @@ class RequestParams extends PagingAndSortingCriteriaAdapter
      * Selection mode for the specified organisation units, default is
      * ACCESSIBLE.
      */
-    private OrganisationUnitSelectionMode ouMode;
+    private OrganisationUnitSelectionMode ouMode = OrganisationUnitSelectionMode.DESCENDANTS;
```

Comment:
Why do we assume the default is DESCENDANTS?

Reply:
That is a good question but I am leaving the answer to the ACL epic.
I just left everything as it was before in this PR